### PR TITLE
Remove map transformations

### PR DESCRIPTION
**Pull request by oysand**

### CODE REVIEW

File: src/isar/mission_planner/echo_planner.py [hunk 1]
```diff
@@ -123,20 +120,15 @@ def _get_robot_pose(self, tag_name: str) -> Pose:
         predefined_pose: Pose = predefined_poses[tag_name]
 
         if predefined_pose.frame is Frame.Robot:
+            # TODO: Should we raise an error and require workaround pose to be asset frame?
```
Comment (oysand) [inline src/isar/mission_planner/echo_planner.py:123]: Should we raise and error and require the workaround poses to be asset frame?

Comment (aestene) [reply src/isar/mission_planner/echo_planner.py:123]: Yes.

File: src/isar/services/coordinates/transformation.py [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import logging
 from typing import Optional
 
-from alitra import AlignFrames, FrameTransform, Point, Quaternion
+from alitra import AlignFrames, FrameTransform, MapConfig, Point, Quaternion
```
Comment (aestene) [inline src/isar/services/coordinates/transformation.py:4]: Why not remove this file entirely?

**Review by aestene — CHANGES_REQUESTED** [review]
One comment on a file I think could be removed.

Would like to implement this for `isar-turtlebot` and check that it works.

**Review by aestene — APPROVED** [review]
LGTM

**Pull request merged**